db/versions/0004.yml

```diff
@@ -0,0 +1,19 @@
+version: 4
+# in version 3, this table was incorrectly set up with timestamp columns
+# lacking timezones, causing issues when the server and client timezones do not
+# agree.
+migrationScript: |-
+  begin
+    alter table azure_queue_messages
+      alter column inserted type timestamptz,
+      alter column visible type timestamptz,
+      alter column expires type timestamptz;
```

Comment:
Do we need to set the timestamp globally? Some tables in entities have `expires` as a column. I wonder if a scan with a date condition would create issues for the entities.

Reply:
I think that's OK because it's in a JSON object that we both encode and decode in Node.